### PR TITLE
[Docs] Improve e2e testing documentation

### PR DESCRIPTION
**Pull request by jhoward1994**
### What does it do?

Improve the e2e testing docs to reflect the latest changes

### CODE REVIEW

File: docs/docs/testing/e2e/app-template.md [hunk 1]
```diff
@@ -43,4 +43,54 @@ This endpoint `DELETES` every row from every table _excluding_ the "core" tables
 
 #### Why do we have it?
 
-This lets us wipe the entire test instance _if_ we need to. DTS does technically does this already for us. But nonetheless, its useful.
+This lets us wipe the entire test instance _if_ we need to. DTS does technically
+does this already for us. But nonetheless, its useful.
```
Comment (ronronscelestes) [inline docs/docs/testing/e2e/app-template.md:47]: small typo
```suggestion
DTS technically does this already for us. But nonetheless, its useful.
```

File: docs/docs/testing/e2e/app-template.md [hunk 2]
```diff
@@ -43,4 +43,54 @@ This endpoint `DELETES` every row from every table _excluding_ the "core" tables
...
+  await page.request.fetch('/api/config/ratelimit/enable', {
+    method: 'POST',
+    data: { value: enabled },
+  });
+}
+```
+
+##### What does it do?
+
+This endpoint can be used to enable or disable to rate limitting middleware in
```
Comment (ronronscelestes) [inline docs/docs/testing/e2e/app-template.md:68]: not sure if it's just me but 
`can be used to enable or disable to rate limitting middleware`
sounds a bit strange to me, wdyt?

File: docs/docs/testing/e2e/data-transfer.md [hunk 3]
```diff
@@ -49,21 +45,38 @@ If you change any of the content schemas (including adding new ones) be sure to
...
+Once this has been done, add the `.tar` backup to `/e2e/data` so the helper
+functions can import it correctly.
+
+As our suite of e2e tests grows we may hold more DTS backups in order to restore
+the Strapi application to a desired state prior to testing.
 
 ### Importing in test scenarios
 
-There's an abstraction for importing the data programtically during tests named `resetDatabaseAndImportDataFromPath` found in `e2e/scripts/data-transfer.js`. Typically, you'll want to run this **before** each test:
+There's an abstraction for importing the data programtically during tests named `resetDatabaseAndImportDataFromPath` found in `e2e/scripts/dts-import.js`. Typically, you'll want to run this **before** each test:
```
Comment (ronronscelestes) [inline docs/docs/testing/e2e/data-transfer.md:75]: small typo
```suggestion
There's an abstraction for importing the data programmatically during tests named `resetDatabaseAndImportDataFromPath` found in `e2e/scripts/dts-import.js`. Typically, you'll want to run this **before** each test:
```

**Review by ronronscelestes — COMMENTED** [review]
overall looks good, thank you!

**Review by joshuaellis — APPROVED** [review]
Apart from @ronronscelestes's suggestions – looks good to me :)

**Pull request merged**